pkg/config/config.go

```diff
@@ -150,12 +150,12 @@ func (c *Config) IsEventMatch(eventType string, jsonEventData interface{}) (bool
 				}
 
 				if value == event.JSONPath.Match {
-					return true, &action
+					return true, &action, index
 				}
 			}
 		}
 	}
-	return false, nil
+	return false, nil, -1
```

Comment:
What does -1 mean? If we use magic number and assign value to them please document them using a constant